README.md

```diff
@@ -71,24 +71,6 @@ because in the future Rustfmt might work on code where it currently does not):
   fixes to break our stability guarantees).
 
 
-## Installation
-
-```sh
-rustup component add rustfmt
-```
```

Comment:
I think it's probably worth leaving this section in still given the different rustup profiles. Thoughts?

Reply:
ah, dueling review comments sorry. it is somewhat duplicative isn't it, think i'm fine removing it for now then. if we find folks are really struggling without a specific "install" section then we can add it back/do something different